tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -347,6 +360,30 @@ describe('Target aggregates', () => {
       await targetAggregatesPage.expectTargetDetails(target);
     });
 
+    it('should filter aggregates by place', async () => {
```

Comment:
Can you also add steps for this scenario to change and validate the selected place?